### PR TITLE
E-ICO: add module

### PR DESCRIPTION
**Pull request by Page-MS**
Also add some other empty pages for other modules linking.

### CODE REVIEW

**Review by maxencevahedi — CHANGES_REQUESTED** [review]
A few questions, will commit some changes soon :)

File: docs/E-ICO.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+
+
+Let’s investigate that by taking a look at the most obviously broken component in Jami : the username info box tooltip (the small i on the right)
+
+![Screenshot of the Jami account creation page showcasing the discussed tooltip](ressources/E-ProgrammingInclusiveAndAccessibleSoftware/JamiToolTip.png)
+
+It’s an “info box” (as called in the code), but a screen reader will handle it as a push button. That seems weird, as this is used as a tooltip, not a button accomplishing an action. Additionally, it doesn’t react well when selecting and unselecting the username text edit.
+
+That is because in the code, this component is indeed a button. That goes without saying that it’s not a good way of implementing it and as a result the corresponding code is very extensive (more than 80 lines of code)! This is the cause of both the accessibility and the selection issue. While it technically works(barely) their are plenty of ways to use default components for a better result. In this case, broken accessibility indicate bad implementation choices.
```
[the remark this answers is not in the document]
Comment (Page-MS) [reply docs/E-ICO.md]: "it doesn’t react well when selecting and unselecting the username text edit."

File: docs/E-ICO.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+That said, a few things can be set to try and automatically detect as much issue as possible. If your app is a web-view, this will be very easy and convenient. You can go to the [E-ETD Accessible technologies Desktop](E-ATD.md) module for more information.
+
+If you app is native, you can set-up your CI to detect missing accessibility labels on focusable elements and setup a test that navigate through all od the elements of a view using the keyboard, checking that none of them wasn’t reached.
+
+You can also use a template of a checklist containing the main elements to be sounded and share it with your team. We provide you, our own accessibility checklist template. Feel free to iterate upon it and to propose your enhancements in this repository.
+
+![Diagram showing an "A very basic accessibility checklist not to break anything when committing. This diagram is available in text format further down this page."](ressources/E-ProgrammingInclusiveAndAccessibleSoftware/InclusivenessChecklist.png)
+
+**The diagram in text format:**
```
[the remark this answers is not in the document]
Comment (Page-MS) [reply docs/E-ICO.md:167]: I preferred the more complete version but feel free to change it if you wish to

File: docs/E-ICO.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+```cpp
+Keys.onUpPressed: verticalScrollBar.decrease()
+Keys.onDownPressed: verticalScrollBar.increase()
+```
+
+By reimplementing a scroll feature and binding it to the arrows key, we effectively broke the base behavior of the component. Here, broken accessibility showcase flawed design as the expected behavior of the component hasn’t been thought all the way through.
+
+In this case, the fix was rather simple, use the base component rather the custom one.
+
+### 1.C Focus and explicit key navigation, choose the right tool at the right time (desktop)
```
[the remark this answers is not in the document]
Comment (maxencevahedi) [reply docs/E-ICO.md]: Then i'll add a section for mobile - but i fear there are a lot of things to say, we'll see when writing it if it holds in this module.
Do you have other things to add that are common to desktop platforms ? I feel like there could be some more explanation on how desktop OSes typically handle accessibility but that's from an outside perspective.

**Review by maxencevahedi — CHANGES_REQUESTED** [review]
Some more things to do.
Will do the mobile section asap!

File: docs/E-ICO.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (maxencevahedi) [reply docs/E-ICO.md]: Are there not platform specifities between Linux and Windows on how to typically navigate and use a screen reader ? That pretty much equals to comparing Orca against NVDA but if the rationale is the same there is indeed no point in doing it.
My intuition comes from Android and iOS that share not so many design principles and usages when it comes to a11y.

[the remark this answers is not in the document]
Comment (maxencevahedi) [reply docs/E-ICO.md]: Besides, should 1.C Focus and explicit key navigation, choose the right tool at the right time (Desktop) be more on the design side ? I feel like it is a design decision really.

**Comment by maxencevahedi** [comment]
Possibility :
- 2 Development pipeline becomes "Accessibility checks"
- 2.1 becomes a new module from a management perspective : how to work work between the teams (design, product, dev etc.) useful for managers as they also need to be guided if we want a11y to be a subject in there structure. We need at some point to talk big picture about the pipeline and how the teams are going to be coordinated - not just the devs
- 2.2 becomes 2.1, the diagram becomes 2.2

File: docs/E-ICO.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Page-MS) [reply docs/E-ICO.md]: - module development pipeline
- not listen blindly to designers

File: docs/E-ICO.md [hunk 4]
```diff
@@ -131,21 +131,10 @@ KeyNavigation.down: KeyNavigation.tab
...
-
-The coach of different teams can meet regularly to coordinate and update the accessibility roadmap of the application. They also can organize monthly meetings with the all of their teams to report on the improvements, blockers and perspectives. The main goal is to normalize accessibility and inclusiveness as routine parts of development.
+## 2. Development process
 
-We highly recommend creating a template of issues for accessibility and encouraging users and developers to fill them when encountering an issue. This would make it easier for developers to pic up an accessibility issue in the backlog and fix it.
+### 2.1 What can be automated?
```
Comment (maxencevahedi) [inline docs/E-ICO.md:136]: If consistent this would be 2.A - consider switching the other to 2.1,2,3 or to switch this to 2.A,B...
